build.rs

```diff
@@ -130,6 +130,7 @@ const RING_X86_64_SRC: &'static [&'static str] =
       "crypto/bn/asm/x86_64-mont.pl",
       "crypto/bn/asm/x86_64-mont5.pl",
       "crypto/chacha/asm/chacha-x86_64.pl",
+      "crypto/cipher/asm/chacha20_poly1305_x86_64.pl",
```

Comment:
This file isn't actually part of the build yet. If having this file sitting around being unused causes trouble, then add a commit that deletes the file.